### PR TITLE
Dynamically change count of selected Program or National Standard Type

### PR DESCRIPTION
**Pull request by littleforest**
Asana ticket: https://app.asana.com/0/1203289004376659/1204601802991345/f

This dynamically changes the count of the number of Program Type checkboxes checked and the National types checked. The Clear All is using a native Custom Event instead of a Stimulus method. I've tried unsuccessfully to use the Stimulus [dispatch](https://stimulus.hotwired.dev/reference/controllers#cross-controller-coordination-with-events) method or the Stimulus [outlet feature](https://stimulus.hotwired.dev/reference/outlets) for cross-controller communication. Would like to get this out now and can revisit the Custom Event code later.

[Reference used for implementing the Custom Event](https://fullstackheroes.com/tutorials/stimulus/create-custom-events/)

https://github.com/ApprenticeshipStandardsDotOrg/ApprenticeshipStandardsDotOrg/assets/1938665/e4b4b22c-f297-4462-bfa6-5b56c96b8ac4

### CODE REVIEW

File: app/assets/stylesheets/application.css [hunk 1]
```diff
@@ -18,44 +18,42 @@
  @tailwind components;
  @tailwind utilities;
 
-
-
  .occupation-tabs *:focus, .occupation-tabs input:focus + label {
```
Comment (FerPerales) [inline app/assets/stylesheets/application.css:21]: We may not need this change as part of this PR

Note: it seems we don't have an standard regarding indentation. We should set something to handle that now that there is five people in the project

File: app/javascript/controllers/checkbox_filter_controller.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+  }
+
+  updateCount() {
+    const length = this.inputTargets.filter(checkbox => checkbox.checked).length
+    this.setCountValue(length)
+  }
+
+  setCountValue(length) {
+    this.countTarget.innerHTML = length
+    if (length > 0) {
```
Comment (FerPerales) [inline app/javascript/controllers/checkbox_filter_controller.js:18]: We can use [toggle](https://developer.mozilla.org/en-US/docs/Web/API/Element/classList):

```
this.countWrapperTarget.classList.toggle(this.displayableClass, length > 0)
```

**Comment by FerPerales** [comment]
@littleforest looks good. Left a couple of comments

**Comment by fatimaburke** [comment]
All good on the design end.

**Review by fatimaburke — APPROVED** [review]

File: app/assets/stylesheets/application.css (hunk 1 above)
Comment (littleforest) [reply app/assets/stylesheets/application.css:21]: I can remove the indentation change.

File: app/javascript/controllers/checkbox_filter_controller.js (hunk 2 above)
Comment (littleforest) [reply app/javascript/controllers/checkbox_filter_controller.js:18]: Oh cool, I didn't know toggle could take an argument like that.

**Pull request merged**